core/src/main/java/com/redhat/lightblue/client/response/DefaultLightblueResponse.java

```diff
@@ -141,6 +141,10 @@ private int parseInt(String fieldName) {
         return field.asInt();
     }
 
+    public JsonNode parseProcessed() {
+        return json.path("processed");
```

Comment:
I'd rename this method to getProcessed(), no parsing is done here.

Also, it might make more sense to use json.get('processed"), as that will return null if "processed" does not exist.